File: java/eclipse/archive/Iteration/src/Iteration.java
public class Iteration {
	
	public static void main(String[] args) {
		int[] mjr = {1, 2, 3, 4};
		
		for(int i = 0; i < mjr.length; i++) {
			System.out.println(mjr[i]);
		}
		
		for(int i : mjr) {
			System.out.println(i);
		}
		
		int i = 0; 
		while (i < mjr.length) {
			System.out.println(mjr[i]);
			i++;
		}
		
		i = 0; 
		do {
			System.out.println(mjr[i]);
			i++; 
		} while(i < mjr.length);
		
		
	}
}

File: python/anaconda/spyder/packages/numpy/iteration.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Tue Jun 25 20:25:29 2019

@author: mrodgers
"""

import numpy as np

mjr = [[1, 2, 3],
       [4, 5, 6]]

mjr = np.array(mjr)

for i in mjr:
    for j in i:
        print(j)
        
for i in range(mjr.shape[0]):
    for j in range(mjr.shape[1]):
        print(mjr[i, j])
        print(mjr[i][j])
        
i, j = mjr.shape
